packages/shared/src/modules/users/libs/validation-schemas/user-profile.validation-schema.ts

```diff
@@ -3,8 +3,8 @@ import { z } from "zod";
 import { UserValidationMessage, UserValidationRule } from "../enums/enums.js";
 
 type UserProfileRequestValidationDto = {
-	firstName: z.ZodString;
-	lastName: z.ZodString;
+	firstName?: z.ZodOptional<z.ZodString>;
+	lastName?: z.ZodOptional<z.ZodString>;
```

Comment:
```suggestion
	firstName: z.ZodOptional<z.ZodString>;
	lastName: z.ZodOptional<z.ZodString>;
```
it can not be optional
it should be string or null